### DISCUSSION TITLE
Conf files calling other conf files?

### DISCUSSION THREAD

**Discussion by raekuul** [joncampbell123/dosbox-x#5187] [Ideas]
Right now the way to call multiple conf files is to call them in sequence from the command line, e.g. `dosbox-x -conf general.conf -conf specific.conf`.  Is it possible for specific.conf to tell dosbox-x that it needs to load general.conf first?  That way I can write smaller specific conf files and just say `require general.conf` for the settings that should be the same across the board, while still only needing to invoke one conf file from the command line (for ease-of-use).

**Comment by Barr-find** [comment]
Or, add a dual .conf file system where we have default.conf and a second file like user.conf. The second file would then override options found in default.conf.
I just updated my portable version of DOSBox-X and it completely overrode my handcrafted config. Now I have to spend hours fixing the damage.

To clarify, I don't mean "let the user choose," because you already have that feature and it doesn't autoload.
<img width="1842" height="1202" alt="image" src="https://github.com/user-attachments/assets/e04540e3-4948-47fa-9b89-3de8222681fb" />

I mean automagicaly:
<pre>If file(".\user.conf").exists == TRUE "load and use that file"</pre>
